.storybook/webpack.config.js

```diff
@@ -42,16 +42,13 @@ module.exports = (baseConfig, env, config) => {
           loader: 'babel-loader',
           options: {
             babelrc: false,
+            configFile: `${__dirname}/.babelrc`,
             // Don't use the production environment as it contains optimisations
             // that break compilation. The shopify/react preset enables the
             // babel-plugin-transform-react-constant-elements plugin which
             // somehow hoists things up into an undesirable location.
-            forceEnv: isProduction ? 'not-production' : undefined,
+            envName: isProduction ? 'not-production' : undefined,
```

Comment:
Does compilation still break on Heroku if this workaround is removed?

Reply:
Heroku doesn't break as that runs the storybook dev server (we talked about switching that to a production build yesterday).

It sill breaks the production build for the reason in the comment.

Reply:
Ok! I was wondering if the upgrades in this PR would have magically fixed it.